### PR TITLE
Add timestamp

### LINKED ISSUE #13: Include timestamp of the exception
Include the time when the exception was raised in the notification.

### PR DESCRIPTION
**Pull request by brunvez**
* Added timestamp to the `ErrorInfo` struct
* Added timestamp to the email mailer text and HTML content
* Added timestamp to webhook notifier

Closes #13

### CODE REVIEW

**Review by iaguirre88 — COMMENTED** [review]
Great work and thanks for your contribution 💪 ❤️ 

At first sight it looks great, but I think it would be nice to add the timestamp in the webhook notifier too. Let me know if you are ok with this or if you have any doubts

I promise I'll make a deeper review tomorrow

**Comment by brunvez** [comment]
Nice catch, I didn't notice that one. Should be updated now.

Sure take your time 😉

**Review by iaguirre88 — APPROVED** [review]
As I mentioned before, great work!

I hope you keep contributing to the project and we are more than welcome to help if you need a hand or have some questions about any another issue in the repo 

Don't hesitate to reach us out

File: lib/boom/mail_notifier/html_content.ex [hunk 1]
```diff
@@ -42,4 +44,8 @@ defmodule Boom.MailNotifier.HTMLContent do
       line: line
     }
   end
+
+  defp format_timestamp(timestamp) do
+    timestamp |> DateTime.truncate(:second) |> DateTime.to_string() |> String.trim_trailing("Z")
```
[the remark this answers is not in the document]
Comment (brunvez) [reply lib/boom/mail_notifier/html_content.ex]: Yeah it's for developers so I agree we can leave it

[the remark this answers is not in the document]
Comment (iaguirre88) [reply lib/boom/mail_notifier/html_content.ex]: Oh, sorry I wasn't clear enough :sweat_smile: 

What I meant is that in Elixir there are two modules for dealing with dates. `DateTime` and `NaiveDateTime`.
The former handles timezones whereas the latter doesn't.

So, there are two ways to remove the `Z` at the end (and I agree with you it was ok to remove it):
- One is the way you did id: using `DateTime` and `String.trim_trailing("Z")`
- Another way is using `NaiveDateTime` which does not include the `Z` by default

For me, the way you did it before was just fine and that is why I approved it. Feel free to revert your last commit and I'll try to get it merged by this week.

Thanks again and sorry for the miscommunication

[the remark this answers is not in the document]
Comment (brunvez) [reply lib/boom/mail_notifier/html_content.ex]: Gotcha, should be alright now 😅 . Feel free to add more comments!

**Review by iaguirre88 — APPROVED** [review]

**Pull request merged**